.mockery.yaml

```diff
@@ -0,0 +1,10 @@
+with-expecter: True
```

Comment:
By default, this option is already `true`.
```suggestion
```

Reply:
updated